apps/dex/src/compounds/ExportModal/ExportModal.tsx

```diff
@@ -172,41 +173,43 @@ const ExportModal = (props: ModalProps & { denom: string }) => {
               (event) => setAmount(event.target.value),
               [],
             )}
+            leadingIcon={
+              <div className="flex gap-1.5 pl-1.5">
+                <Label
+                  type="button"
+                  onClick={useCallback(() => {
```

Comment:
out of curiosity, is inline handlers a coding style recently? I do see a mix of it in sifchain's codebase, like inline vs handleClickXYZ

Reply:
I think we should start isolating this kind of handlers in custom hooks for better code scalability. something like:

```tsx

const { state, actions } = useXYZ()

// ...

return (
  <div>
    <button onClick={actions.doStuff}>do stuff</button>
  </div>
)

```

Reply:
Yeah pretty much this is an active development solution before the implementation is finalized n stable, then I would extract it out to a reusable hook, much easier to do development iterations quickly while it's inline.

Also for shorter stuff like `useCallback(() => setValue("blah"), [])` I've been throwing it inline, it's a judgement call really :v